amundsen-kube-helm/templates/helm/amundsen/templates/service.yaml

```diff
@@ -40,7 +40,7 @@ spec:
   ports:
     - port: 5002
       name: {{ .Chart.Name }}-{{ .Values.metadataServiceName }}-{{ .Values.environment }}-http
-      targetPort: 5000
+      targetPort: 5002
```

Comment:
did we need to change the port for search service as well given we change it to 5001 for default?

Reply:
oh if that was changed then yes. I guess I haven't pulled in the newest search service version yet. I will push that change too.